client/src/components/common/limitation/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import useStyles from './style';
+
+export default function CircularIndeterminate({
```

Comment:
1- use Es6 in writing your component & I prefer to call the function by folder's name , if any one of them should be the same all the time , so no need to pass it as an argument & you can but it in `jsx` only, but for example, if we need to edit or update on style rather than, style from the component it self so we can pass another classname beside the main one , so for example: put `bodyClass` as static cause it comes from Library already & pass beside it new class name from your creation and add this style in our file then pass it as props . 
2- don't forget to add props & default props also to your file. think about it deeply . 
```
const Limitation =({ bodyClass, circulClass , circulSize })
```